### PR TITLE
Feature/add email field to event

### PR DESCRIPTION
**Pull request by emmadeno**

### CODE REVIEW

File: app/src/main/java/com/ncnf/Utils.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
     public static final String INVITED_KEY = "invited";
     public static final String TAGS_LIST_KEY = "tags";
     public static final String ADDRESS_KEY = "address";
+    public static final String EVENT_EMAIL_KEY = "email";
```
Comment (eloigrndl) [inline app/src/main/java/com/ncnf/Utils.java:43]: This key already exists (EMAIL_KEY)

**Review by khcr — APPROVED** [review]
Small but necessary change

**Review by eloigrndl — CHANGES_REQUESTED** [review]
Good job ! But you must change the key of e-mail

File: app/src/main/java/com/ncnf/Utils.java (hunk 1 above)
Comment (emmadeno) [reply app/src/main/java/com/ncnf/Utils.java]: done :)

**Review by eloigrndl — APPROVED** [review]

**Pull request merged**